rubocop-bitcrowd.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |spec|
   spec.bindir        = 'exe'
   spec.executables   = 'rubocop-autofix'
 
-  spec.add_runtime_dependency 'rubocop', '>= 0.56'
+  spec.add_runtime_dependency 'rubocop', '~> 0.56'
```

Comment:
using `>=` results in the following warning when building the gem:

```
> gem build rubocop-bitcrowd.gemspec
WARNING:  open-ended dependency on rubocop (>= 0.56) is not recommended
  if rubocop is semantically versioned, use:
    add_runtime_dependency 'rubocop', '~> 0.56'
WARNING:  See http://guides.rubygems.org/specification-reference/ for help
```

Reply:
Ok, just learned, the thing is called
> twiddle-wakka

Makes perfect sense 🖤